MainModule/Client/UI/Default/UserPanel.lua

```diff
@@ -243,13 +246,14 @@ return function(data, env)
 				})
 
 				entryText = entryBox:Add("TextBox", {
-					Position = UDim2.new(0, 55, 0, 10);
-					Size = UDim2.new(1, -60, 0, 25);
+					Position = UDim2.new(0.043, 0, 0.324, 0);
+					Size = UDim2.new(0.928, 0, 0.238, 0);
 					Text = "";
 					PlaceholderText = "Type entry here";
```

Comment:
would this text be too confusing for the vast majority of roblox players?

Reply:
I dont know. Not my Problem. Wont Fix,